services/sensu/service.go

```diff
@@ -55,18 +55,20 @@ func (s *Service) Update(newConfig []interface{}) error {
 }
 
 type testOptions struct {
-	Name   string      `json:"name"`
-	Source string      `json:"source"`
-	Output string      `json:"output"`
-	Level  alert.Level `json:"level"`
+	Name     string      `json:"name"`
+	Source   string      `json:"source"`
+	Output   string      `json:"output"`
+	Handlers []string    `json:"handlers"`
+	Level    alert.Level `json:"level"`
 }
 
 func (s *Service) TestOptions() interface{} {
 	return &testOptions{
-		Name:   "testName",
-		Source: "Kapacitor",
-		Output: "testOutput",
-		Level:  alert.Critical,
+		Name:     "testName",
+		Source:   "Kapacitor",
+		Output:   "testOutput",
+		Handlers: []string{"testHandlers"},
```

Comment:
By default I think it would be best to have an empty list of Handlers. Otherwise a default call to test the sensu service will probably fail.